### DISCUSSION TITLE
Text input disabled on iOS PWA

### DISCUSSION THREAD

**Discussion by coder543** [open-webui/open-webui#7249] [Issues]
# Bug Report

I'm using the latest Open WebUI through Docker (v0.4.3), and while it works fine from Safari, if I add this to Home Screen and launch it, text input does not work, period. It is impossible to bring up the touchscreen keyboard to sign in, and even attaching a bluetooth keyboard does not solve the issue.

I'm sorry that I deleted the rest of the template, but... there's really not much more to say than that. I have tried removing and re-adding the PWA to the home screen, and nothing has resolved it. If this issue is not specific to my phone, it should be extremely easy for anyone with an iPhone to reproduce... because you can't even sign into the PWA.

**Comment by tompetk** [comment]
Did reproduce this on iOS 18.1.1, only phone restart helped.

**Comment by tkafka** [comment]
I just got this in 18.2.1 with a latest open-webui 0.5.4 and it was fixed by restart. Seems like a Safari bug :/.
